containers/kubernetes/reference-content/exposing-services.mdx

```diff
@@ -0,0 +1,75 @@
+| **Node Port Service**  | • Exposes a specific port on each node of a cluster. <br/>• Forwards external traffic received on that port to the right pods. | • Exposing single-node, low-traffic clusters to the internet for free <br/>• Testing. | • Not ideal for production or complex clusters <br/> • Single point of failure.<br/> • Not all port numbers can be opened.|
+| **Load Balancer Service** | • Creates a single, external Load Balancer with a public IP <br/> • External LB forwards all traffic to the corresponding LoadBalancer service within the cluster <br/> • LoadBalancer service then forwards traffic to the right pods. <br/> • Operates at the L4 level | • Exposing a service in the cluster to the internet. <br/>• Production envs (highly available). <br/>• Dealing with TCP traffic. | • Each service in the cluster needs its own Load Balancer (can become costly) |
+| **Ingress** | • A native resource inside the cluster (not a service) <br/>• Ingress Controller receives a single, external public IP, usually in front of a spun-up external HTTP Load Balancer <br/> • Uses a set of rules to forward web traffic (HTTP/S) to the correct service out of multiple services within the cluster. <br/> • Each service then sends the traffic to a suitable pod. <br/> • Operates at the L7 level.| • Clusters with many different services <br/> • Dealing with HTTP/S traffic | • Requires an Ingress Controller (not included by default, must be created). <br/> • Designed for HTTP/S traffic only (more complicated to configure for other protocols). | 
+
+
+Our [webinar](https://www.youtube.com/watch?v=V0uKqYXJRF4) may also be useful to you when considering how to expose your cluster. From 5m47 to 13m43 the different methods to expose a cluster are described and compared.
+
+## NodePort Service
+
+<Lightbox src="scaleway-nodeport-diag.webp" alt="" />
+
+For more information and practical help with creating a NodePort Service, check out the following resources:
+
+- [Exposing a service via NodePort - Tutorial](/tutorials/get-started-deploy-kapsule/#exposing-the-service-for-testing-via-nodeport-(optional))
+- [Exposing a service via NodePort - Video demonstration](https://www.youtube.com/watch?v=R5CwAq1a0Og) (from 23m52)
+- [Comparison of NodePort vs LoadBalancer](/tutorials/get-started-kubernetes-loadbalancer/#why-do-we-need-a-load-balancer)
+
+## LoadBalancer Service
+
+<Lightbox src="scaleway-lb.webp" alt="" />
+
+For more information and practical help with creating a LoadBalancer Service, check out the following resources:
+
+- [Creating and configuring a Load Balancer service](/containers/kubernetes/reference-content/kubernetes-load-balancer)
+- [Getting started with Kubernetes Load Balancers - Tutorial](/tutorials/get-started-kubernetes-loadbalancer/)
+- [Getting started with Kubernetes Load Balancers - Video demonstration](https://www.youtube.com/watch?v=R5CwAq1a0Og)
+
+## Ingress
+
+<Lightbox src="scaleway-ingress.webp" alt="" />
```

Comment:
@RoRoJ when clicking the picture to display it bigger, the new picture is actually not that bigger (almost the same size). It's not really easy to read then.